DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -269,6 +270,56 @@
 mu8diEle12CaloIdLTrackIdL_dz.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_DiEle12_CaloIdL_TrackIdL_DZ_v*")
 mu8diEle12CaloIdLTrackIdL_dz.denGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_Mu8_DiEle12_CaloIdL_TrackIdL_v*")
 
+###########AK8 jet trigger with AK8 CSV requirement############
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring = hltBTVmonitoring.clone()
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.FolderName = cms.string('HLT/Higgs/ggHbb/CSV04/')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.nmuons = cms.uint32(0)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.nelectrons = cms.uint32(0)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.njets = cms.uint32(1)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.jets = cms.InputTag("ak8PFJetsCHS")
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.jetSelection = cms.string('pt>300 & abs(eta)<2.4')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.bjetSelection = cms.string('pt>330 & abs(eta)<2.4')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_AK8PFJet330_PFAK8BTagCSV_p1_v*')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.histoPSet.jetPtBinning = cms.vdouble(0,300,310,330,350,370,400,450,500,700,1000,1500,3000)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.histoPSet.lsPSet = cms.PSet(
+  nbins = cms.uint32( 1 ),
+)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.histoPSet.htPSet = cms.PSet(
+  nbins = cms.uint32( 0 ),
+  xmin  = cms.double(   0   ),
+  xmax  = cms.double(   1   ),
+)
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.uint32( 70 ),
+  xmin  = cms.double(   300   ),
+  xmax  = cms.double(  1000 ),
+)
+
+
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring = hltBTVmonitoring.clone()
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.FolderName = cms.string('HLT/Higgs/ggHbb/CSV03/')
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.nmuons = cms.uint32(0)
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.nelectrons = cms.uint32(0)
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.njets = cms.uint32(1)
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.jets = cms.InputTag("ak8PFJetsCHS")
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.jetSelection = cms.string('pt>300 & abs(eta)<2.4')
+AK8PFJet330_TrimMass30_CSV04_PromptMonitoring.bjetSelection = cms.string('pt>330 & abs(eta)<2.4')
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_AK8PFJet330_PFAK8BTagCSV_p17_v*')
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.histoPSet.jetPtBinning = cms.vdouble(0,300,310,330,350,370,400,450,500,700,1000,1500,3000)
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.histoPSet.lsPSet = cms.PSet(
+  nbins = cms.uint32( 1 ),
+)
+AK8PFJet330_TrimMass30_CSV03_PromptMonitoring.histoPSet.htPSet = cms.PSet(
```

Comment:
I'm a little bit confused
what does it mean #bins = 0 ?

Reply:
When I ran the code the #bins in the histogram was 1 instead of 0, but I can change that to 1 so it's not confusing

Reply:
I'm wondering if it runs w/ #bins = 0

Reply:
Yes it ran when the ht #bins = 0.